### ISSUE TITLE
Big files (>1gb) are breaking uploads

### ISSUE THREAD

**Issue by felipegaucho** [Paratii-Video/paratii-portal#819]
When uploading big files (trying a 1.6gb one), our progress bar gets to 50% and stops (local upload *seems* to complete). We get the "preparing your file" notification, but not the "Upload done, let's start transcoding" one. Saving data does not work, and both "Save" and "Publish" buttons get inactive/unclickable after 1st time they're clicked. What I get right after "a complete" local upload:

<img width="790" alt="screen shot 2018-10-03 at 2 51 40 am" src="https://user-images.githubusercontent.com/26517120/46394728-f6feab00-c6b7-11e8-9089-6930cbfc7a99.png">

On a rare case when I managed an upload of this magnitude "till the end", playback doesn't work - I get a black screen, "hls:4" error code inside - "manifestLoadError" in the console. The gateway doesn't find the file manifest, and, when trying to fetch the hash directly from the gateway I get "reply interface called twice".

Like: https://lab.paratii.video/play/YoPXLJqQbU31

This is consistent with behaviour @jellegerbrandy has reported. Focusing on the upload of the big file itself (data saving prob. is likely ParatiiJS waiting for something that never happens):

- Maybe it has to do w/ something used to enforce our old size limitation?
- Maybe there's some time limit these big guys are surpassing?
- Clogged webrtc channels (or bad handling of them)?
- Maybe there's a way to streamline that upload?

(small files are working just fine 👌)

**Comment by jellegerbrandy** [comment]
Yeah, just for further clarification there are really two different issues at play here:

(1) problems with uploading/transcoding, where the problem seem to mostly lie in the communication channel/limited browser resources. This seems browser related: I do not have these isseus when uploading/transcoding using a node script (i.e. not using the browser).

(2) problems with the resulting ipfs hash:  "reply interface called twice"  (in the case of felipes example above it is this one: https://gateway.paratii.video/ipfs/QmVGpv1aeAiFEECq19jYDTJJh9HZGR1zSnc8oH41kZ6B7r/).

@ya7ya ? :-)
